### PR TITLE
Change the text of a jQuery challenge description

### LINKED ISSUE #14608: “Every HTML element is a child of your body element”
Challenge [Target the Children of an Element Using jQuery](https://www.freecodecamp.com/challenges/target-the-children-of-an-element-using-jquery) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.1 Safari/603.1.30</code>.

Every HTML element in the body, not those in the head. And guaranteed only to be a descendant, not necessarily a child.

### PR DESCRIPTION
**Pull request by Ntropish**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #14608 

#### Description
Rewrites the description of the challenge "jQuery: Target the Children of an Element Using jQuery".
It doesn't change the challenge itself other than a slight rewording of the instructions.

The reason for the change is that the description was incorrect. It described descendants as if they were children. It also previously talked about property inheritance which is outside the scope of the challenge. Going outside the scope violates the instruction "Each challenge should teach exactly one concept" from the challenge style guide.

### CODE REVIEW

**Comment by frutsy007** [comment]
Let my

File: seed/challenges/03-front-end-libraries/jquery.json [hunk 1]
```diff
@@ -1195,12 +1195,13 @@
...
+        "When HTML elements are placed one level below another they are called <code>children</code> of that element. For example, the button elements in this challenge with the text \"#target1\", \"#target2\", and \"#target3\" are all <code>children</code> of the <code>&#60;div class=\"well\" id=\"left-well\"&#62;</code> element.",
         "jQuery has a function called <code>children()</code> that allows you to access the children of whichever element you've selected.",
-        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color of blue:",
+        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color blue:",
         "<code>$(\"#left-well\").children().css(\"color\", \"blue\")</code>",
-        "Give all the children of your <code>#right-well</code> element a color of orange."
+        "<hr>",
+        "Give all the children of your <code>right-well</code> element the color orange."
+
```
Comment (raisedadead) [inline seed/challenges/03-front-end-libraries/jquery.json]: Remove this extra newline

File: seed/challenges/03-front-end-libraries/jquery.json [hunk 2]
```diff
@@ -1195,12 +1195,13 @@
...
       "description": [
-        "Many HTML elements have <code>children</code> which <code>inherit</code> their properties from their parent HTML elements.",
-        "For example, every HTML element is a child of your <code>body</code> element, and your \"jQuery Playground\" <code>h3</code> element is a child of your <code>&#60;div class=\"container-fluid\"&#62</code> element.",
+        "When HTML elements are placed one level below another they are called <code>children</code> of that element. For example, the button elements in this challenge with the text \"#target1\", \"#target2\", and \"#target3\" are all <code>children</code> of the <code>&#60;div class=\"well\" id=\"left-well\"&#62;</code> element.",
         "jQuery has a function called <code>children()</code> that allows you to access the children of whichever element you've selected.",
-        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color of blue:",
+        "Here's an example of how you would use the <code>children()</code> function to give the children of your <code>left-well</code> element the color blue:",
```
Comment (raisedadead) [inline seed/challenges/03-front-end-libraries/jquery.json]: Consider wrapping `blue` in a `<code></code>`

**Review by raisedadead — CHANGES_REQUESTED** [review]
Some minor changes, and I think this is good.

File: seed/challenges/03-front-end-libraries/jquery.json (hunk 1 above)
Comment (Ntropish) [inline seed/challenges/03-front-end-libraries/jquery.json]: Should it be in the format `[..."lineOne<hr>", "lineTwo"...]`? I'm unfamiliar with the system, thanks for the help.

Comment (raisedadead) [inline seed/challenges/03-front-end-libraries/jquery.json]: @Ntropish Yes, that is correct. The diff shows that this JSON has an extra line. If you are stuck, you can let us know in the [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors), we will try and help.

Comment (raisedadead) [inline seed/challenges/03-front-end-libraries/jquery.json]: Here is how the diff looks to me:
```diff
-        "Give all the children of your <code>#right-well</code> element a color of orange."
+        "<hr>",
+        "Give all the children of your <code>right-well</code> element the color orange."
+
+        ],
+       "challengeSeed": [
```

And here is how it should be:
```diff
-        "Give all the children of your <code>#right-well</code> element a color of orange."
+        "<hr>",
+        "Give all the children of your <code>right-well</code> element the color orange."
+        ],
+       "challengeSeed": [
```

**Comment by raisedadead** [comment]
You can push changes to this same PR by amending your commit, the instructions are there in the [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md).

> git add <file name>
> git commit --amend

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @Ntropish ! Congrats on your first contribution to freeCodeCamp! Sorry about the delay in getting back to you!

Happy coding!

**Pull request merged**